pod/perlop.pod

```diff
@@ -3836,6 +3849,11 @@ Here is a short, but incomplete summary:
 
 Choose wisely.
 
+=head2 Logical or and Exclusive Or
```

Comment:
I believe these were put in the same section because the current organization of perlop is to document operators in the order of precedence, with operators of the same precedence (like or and xor) sharing a section.

Reply:
Is it then the case that there is a distinct precedence order in ``&& || ^^ // ..`` as these all have their own section in this order?  My guess is that ``||`` and ``^^`` have the same precedence.

Reply:
According to the precedence table at the top, `&&` has a distinct precedence and `||` `//` `^^` share a precedence. Perhaps this was an oversight when adding the latter two operators.